src/frontend/XAPI/XAPIStatement.ts

```diff
@@ -136,11 +136,11 @@ export class XAPIStatement {
     } = {
       [ContextExtensionsDefinition.sessionId]: this.sessionId,
     };
-    for (const key of Object.keys(contextExtensions)) {
```

Comment:
This rule does not make sense to me, `for of` seems like the appropriate way to do this kind of stuff.

Reply:
Honestly I never used `for ... of` loop and prefer forEach iterator because ... eslint always forbidden it ^^

Reply:
The reason invoked by eslint is that `for ... of` loop needs `regenerator-runtime` polyfill and it's too heavyweight.